### PR TITLE
Change payload

### PR DESCRIPTION
**Pull request by schung507**
Confirmed this merges user by anonymous_id when user is unidentified

Screen Shot 2023-05-09 at 3 04 45 PM

Testing method:
- Created unidentified click events using heapjs with `anonymous_id`. 
- Sent requests through postman with the same `anonymous_id`
- Make sure user is merged in Heap

_A summary of your pull request, including the what change you're making and why._

## Testing

_Include any additional information about the testing you have completed to
ensure your changes behave as expected. For a speedy review, please check
any of the tasks you completed below during your testing._

- [ ] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [ ] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [ ] [Segmenters] Tested in the staging environment

### CODE REVIEW

File: packages/destination-actions/src/destinations/heap/trackEvent/index.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ type HeapEvent = {
   event: string | undefined
   idempotency_key: string
   timestamp?: string
-  properties: {
+  properties?: {
```
Comment (murphpdx) [inline packages/destination-actions/src/destinations/heap/trackEvent/index.ts:14]: I'm concerned about making changes to the mappings. Also, I think properties should be required. Would the event be useful without them?

Comment (schung507) [reply packages/destination-actions/src/destinations/heap/trackEvent/index.ts:14]: oh it was throwing in the PR, should i just make an empty properties object?

Comment (schung507) [reply packages/destination-actions/src/destinations/heap/trackEvent/index.ts:14]: my payload in testing didnt have properties object

**Comment by murphpdx** [comment]
Looking great, you'll probably need to update the snapshots as well. Did you test using the action tester?

**Comment by murphpdx** [comment]
image

**Pull request closed without merging**